perf/replicache.ts

```diff
@@ -13,23 +13,35 @@ import {
 import {jsonArrayTestData, TestDataObject, jsonObjectTestData} from './data';
 import type {Bencher, Benchmark} from './perf';
 import {range, sampleSize} from 'lodash-es';
-import {makeIdbName} from '../src/replicache';
+import * as kv from '../src/kv/mod';
 import {resolver} from '../src/resolver';
-import {IDB_DATABASES_DB_NAME} from '../src/persist/idb-databases-store';
+import {
+  setupIDBDatabasesStoreForTest,
+  teardownIDBDatabasesStoreForTest,
+} from '../src/persist/mod';
+import {uuid} from '../src/uuid';
 
 export function benchmarkPopulate(opts: {
   numKeys: number;
   clean: boolean;
   indexes?: number;
 }): Benchmark {
+  let repToClose: Replicache | undefined;
   return {
     name: `populate ${valSize}x${opts.numKeys} (${
       opts.clean ? 'clean' : 'dirty'
     }, ${`indexes: ${opts.indexes || 0}`})`,
     group: 'replicache',
     byteSize: opts.numKeys * valSize,
+    async setupEach() {
+      setupIDBDatabasesStoreForTest();
```

Comment:
It seems like all of these tests do the same thing in setup and tear down. I guess it is non trivial to refactor to share code but maybe worth thinking about?

Reply:
They do similar things but not the same.  We have 5 different benchmarks, and actually 3 different setup/teardown processes, for these different setups:
1. shared rep across all runs
2. shared rep across all runs relying on shared persisted state
3. rep per run